docs/src/examples/components/Dropdown/Types/DropdownExampleMultipleSelection.shorthand.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { Dropdown } from '@stardust-ui/react'
+
+const inputItems = [
+  'Bruce Wayne',
+  'Natasha Romanoff',
+  'Steven Strange',
+  'Alfred Pennyworth',
+  `Scarlett O'Hara`,
+  'Imperator Furiosa',
+  'Bruce Banner',
+  'Peter Parker',
+  'Selina Kyle',
+]
+
+const DropdownExampleMultipleSelection = () => (
+  <Dropdown
+    multiple
+    items={inputItems}
+    placeholder="Select your heroes"
```

Comment:
👍  for the placeholder value, was confused with the text on the PR description gif saying 'Start typing a name' as the example is not searchable.